src/main/java/myjournal/Ui.java

```diff
@@ -0,0 +1,80 @@
+package myjournal;
+
+import myjournal.task.Task;
+
+/**
+ * Creates the Ui object.
+ *
+ * @author felissafaustine
+ */
+public class Ui {
+    /**
+     * Prints the welcome message.
+     * @return
```

Comment:
I think you forgot to add the description for `@return` here XD

Reply:
Thanks for pointing it out!! I forgot to change the javadocs after changing my code oops XD